x-pack/plugins/security_solution/public/common/utils/get_mapped_non_ecs_value.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { TimelineNonEcsData } from '@kbn/timelines-plugin/common';
```

Comment:
This is a utility that was part of on of the deleted file but is required at other places as well so moved here.

Reply:
Done in https://github.com/elastic/kibana/pull/196243/commits/8ec47e8a2dbdcb3f5068af2e663814aec3b017a9